functions/go/search-replace/searchreplace/search_replace_test.go

```diff
@@ -106,7 +106,7 @@ func TestDecode(t *testing.T) {
 	if !assert.Error(t, err) {
 		t.FailNow()
 	}
-	expected := `invalid matcher "put-values", must be one of ["by-value" "by-value-regex" "by-path" "put-value" "put-comment"]`
+	expected := `invalid matcher "put-values", must be one of ["by-value" "by-file-path" "by-value-regex" "by-path" "put-value" "put-comment"]`
```

Comment:
where's the unit test exercising file expressions?

Reply:
This is an external library and has been rigorously unit tested https://github.com/bmatcuk/doublestar/blob/master/doublestar_test.go#L29